src/styles/global/form.styl

```diff
@@ -9,15 +9,30 @@
     box-sizing: border-box
     display: block
     padding: .5em 1vw
+    margin-top: 0.5em
 
     &--full-width
       width: 100%
+    &--top-margin
+      margin-top: 0.5em
 
   &__label
-    font-size: .7em
-    font-weight: bold
-    letter-spacing: .1em
-    opacity: .7
+    .org-details-slug-editor__form-input
```

Comment:
it feels strange to me this and `.org-details-slug-editor__form-help` is here in the `global/form.styl`, do these belong in `components/edit-details.styl`? maybe check where else `.form__label` used and leave behind some basic styling.